src/middlewared/middlewared/plugins/docker/state_management.py

```diff
@@ -126,7 +126,7 @@ async def _event_system_ready(middleware, event_type, args):
 
 async def _event_system_shutdown(middleware, event_type, args):
     if await middleware.call('service.started', 'docker'):
-        middleware.create_task(middleware.call('service.stop', 'docker'))
+        middleware.call('service.stop', 'docker')  # No need to wait for this to complete
```

Comment:
I think we should have `await` here as this otherwise just creates an async task otherwise i.e we do `.create_task` in that case. So we won't wait for the job to complete, that will just run